core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -11294,4 +11294,21 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
         isLinux(expectedBigQuery));
   }
 
+  @Test public void testStringAggFuncWithCollation() {
+    final RelBuilder builder = relBuilder().scan("EMP");
+    final RelBuilder.AggCall aggCall = builder.aggregateCall(SqlLibraryOperators.STRING_AGG,
```

Comment:
Can we create a relNode with LIST_AGG WIthin Group instead and have it translated to STRING_AGG at BigQuery.